### ISSUE TITLE
BAM & BED Files

### ISSUE THREAD

**Issue by jfiksel** [etal/cnvkit-examples#4]
Is it possible to access the original BAM and BED files that were used to create the .cnn files? 

Thanks,
Jacob

**Comment by etal** [comment]
Yes, the BAM sources are in dbGaP under the accessions listed in the papers in the [README](https://github.com/etal/cnvkit-examples/blob/master/README.md), and the BED files are in the [intervals](https://github.com/etal/cnvkit-examples/tree/master/intervals) folder in this repo.

**Comment by jfiksel** [comment]
Thanks. I found the targeted & exome accession number, but the cell line C0902 accessions do not seem to be listed anymore in the paper or in the supplemental information.

**Comment by etal** [comment]
The C0902 reads might be on SRA now. If you really need them I can put you in touch with the original source.
